traveler-ios-ui/TravelerKitUI/ModelExtensions/Product+Description.swift

```diff
@@ -0,0 +1,22 @@
+//
+//  Product+Description.swift
+//  TravelerKitUI
+//
+//  Created by Omar Padierna on 2019-06-07.
+//  Copyright © 2019 GuestLogix. All rights reserved.
+//
+
+import Foundation
+import TravelerKit
+
+extension Product {
+    var secondaryDescription: String? {
+        switch self {
```

Comment:
You had added this somewhere else in a different PR...youre gonna get merge conflicts :)

Make sure you rebase and not merge